tutor/src/components/task-plan/footer/delete-link.jsx

```diff
@@ -1,7 +1,40 @@
 import React from 'react';
-import { AsyncButton, SuretyGuard } from 'shared';
+import { observer } from 'mobx-react';
+import { observable, action } from 'mobx';
+import { AsyncButton } from 'shared';
 import Icon from '../../icon';
```

Comment:
Just so you're aware, we've just added https://github.com/openstax/tutor-js/blob/master/tutor/src/helpers/react.js which exports common things.  The above could be: 
`import { React, observer, observable, action } from '../../helpers/react';